mockld/streaming_service.go

```diff
@@ -35,15 +33,14 @@ type StreamingService struct {
 
 type eventImpl struct {
 	name string
-	data map[string]interface{}
+	data interface{}
```

Comment:
Changed how mock stream events are specified so that the data can be any JSON-serializable value or a `json.RawMessage`; that way we can inject more kinds of arbitrary data into the stream.